libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -3629,6 +3629,30 @@ float AP_AHRS::get_air_density_ratio(void) const
     return 1.0 / sq(eas2tas);
 }
 
+void AP_AHRS::set_data_sending_state(bool enabled)
+{
+    switch (active_EKF_type()) {
+#if AP_AHRS_EXTERNAL_ENABLED
+    case EKFType::EXTERNAL:
+        return external.set_data_sending_state(enabled);
+#endif
+    default:
+        break;
+    }
+}
+
+void AP_AHRS::set_gnss_state(bool enabled)
+{
+    switch (active_EKF_type()) {
+#if AP_AHRS_EXTERNAL_ENABLED
+    case EKFType::EXTERNAL:
+        return external.set_gnss_state(enabled);
+#endif
```

Comment:
The code paths that aux function 105 takes could be used to disable GPS in EKF2/3. 
[105](https://ardupilot.org/copter/docs/common-auxiliary-functions.html)

Not required for the PR.

Reply:
Ok, I'll create a next request with these changes

Reply:
I'm not sure that I understood what I need right, but I added the GPS disabling for EKF2/3